kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -104,6 +105,11 @@ def get_highwater_offsets(self, consumer_offsets):
             consumer = self.__create_consumer(consumer_group)
             self.log.debug("Consumer instance %s created for group %s", consumer, consumer_group)
             cluster_metadata = consumer.list_topics(timeout=self.config._request_timeout)
+            try:
+                cluster_id = cluster_metadata.cluster_id
+            except AttributeError:
+                self.log.error("Failed to get cluster metadata for consumer group %s", consumer_group)
+                continue
```

Comment:
How likely is it to get this exception? 

Here we're changing the behaviour of the check if the `cluster_id` is not found because we skip the current consumer_group, which might affect some customers when they will upgrade the agent even though they are not using the new option introduced.

Reply:
I removed the `continue`. Worst case, we will end up with an empty kafka_cluster_id.
This should never happen, but I wanted to be safe

Reply:
Sounds good!